.idea/vcs.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="VcsDirectoryMappings">
+    <mapping directory="$PROJECT_DIR$" vcs="Git" />
+  </component>
+</project>
```

Comment:
Please exclude .idea configuration files from git tracking. If you don't know how - try to google it.

Tip: It's possible to do so by command like that: git rm .idea/ -r --cached